### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards - ExpenseReport]: Talkback does not announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#98]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)

**Repro Steps:**
1. Launch the Adaptive cards app on Android.
2. Turn on Talkback.
3. Navigate to 'Import' control and select it.
4. Navigate to 'Sample cards' and select it.
5. Navigate to available dropdown under 'Sample cards'.
6. Navigate to available dropdown under 'Sample cards' and select it.
7. Navigate all the controls under dropdown and select 'ExpenseReport'.
8. Navigate to 'Show History' and activate it.
9. Observe the issue.

**Actual Result:**
Talkback does not announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'. It just announces "Show History".

**Expected Result:**
Talkback should announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'. It should announce as "Show History button".

**User Experience:**
Screen Reader users will be confused if Talkback does not announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/ae31079d-9262-4f87-86af-5fd89d597007

![Talkback does not announce the role of 'Show History' and 'Hide History' present under 'ExpenseReport'](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/5dae724e-6790-46d4-8d63-9c50df66a5e7)

### Card JSON

```JSON
NA
```

**Comment by MengliElmendorf137** [comment]
The bug is fixed in this PR : https://github.com/microsoft/AdaptiveCards-Mobile/pull/195/commits

**Comment by vagpt** [comment]
This issue is now tracking with #202 as of now it is announcing the role as button but focus is landing twice.
